integrations/ollama/tests/test_generator.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-FileCopyrightText: 2023-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from requests import HTTPError
+
+from ollama_haystack import OllamaGenerator
+
+
+class TestOllamaGenerator:
```

Comment:
let's add one or two unit tests about `__init__`
they can be similar to [these](https://github.com/deepset-ai/haystack-core-integrations/blob/92b9b9be23ed3575a3d50ef527ab6f99056aca54/integrations/cohere/tests/test_cohere_generators.py#L23)